src/store/plugins/sync-camera-lonlatzoom.js

```diff
@@ -45,32 +46,27 @@ export default function syncCameraLonLatZoom(store) {
             store.dispatch('setZoom', { zoom, dispatcher: self })
             store.dispatch('setRotation', normalizeAngle((rotation * Math.PI) / 180))
         }
-        if (mutation.type === 'setCenter' && mutation.payload.dispatcher !== self) {
-            store.dispatch('setCameraPosition', {
-                position: {
-                    x: store.getters.centerEpsg4326[0],
-                    y: store.getters.centerEpsg4326[1],
-                    z: state.position.camera.z,
-                    heading: 0,
-                    pitch: CesiumMath.toDegrees(-CesiumMath.PI_OVER_TWO),
-                    roll: 0,
-                },
-                dispatcher: self,
-            })
-        }
-        if (mutation.type === 'setZoom' && mutation.payload.dispatcher !== self) {
-            const newHeight = calculateHeight(store.getters.resolution, state.ui.width)
-            store.dispatch('setCameraPosition', {
-                position: {
-                    x: state.position.camera.x,
-                    y: state.position.camera.y,
-                    z: newHeight,
-                    heading: 0,
-                    pitch: CesiumMath.toDegrees(-CesiumMath.PI_OVER_TWO),
-                    roll: 0,
-                },
-                dispatcher: self,
-            })
-        }
+    })
+    store.subscribeAction({
+        after: (action, state) => {
+            if (state.position.camera === null) {
```

Comment:
Maybe put a comment here why we use action subscription instead of mutation as we usually do mutation watcher